### PR TITLE
Clone

### PR DESCRIPTION
**Pull request by teckno**

### CODE REVIEW

File: ivy/functional/frontends/torch/other_operations.py [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import ivy
+
+
+def clone(input, *, memory_format=torch.preserve_format):
```
Comment (hirwa-nshuti) [inline ivy/functional/frontends/torch/other_operations.py:4]: This function is not on ivy's functional API

**Review by hirwa-nshuti — CHANGES_REQUESTED** [review]
Hey @teckno 

Thanks for the PR😊
But the function `clone` isn't present in the functional API, also there changes on the `.idea` files would you remove them.

Thanks

File: ivy/functional/frontends/torch/other_operations.py (hunk 1 above)
Comment (teckno) [reply ivy/functional/frontends/torch/other_operations.py:4]: Thanks,

Am I on the right path cause I am following the documentation https://pytorch.org/docs/stable/generated/torch.clone.html
and the video https://www.youtube.com/watch?v=1crW78zKxSc&list=PLwNuX3xB_tv-wTpVDMSJr7XW6IP_qZH0t&index=5&ab_channel=Ivy . Please let me know if am following the correct path.

Thanks again,

File: ivy_tests/test_ivy/test_frontends/test_torch/test_other_operations.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
+# global
+from hypothesis import strategies as st
+
+# local
+import ivy_tests.test_ivy.helpers as helpers
+from ivy_tests.test_ivy.helpers import handle_frontend_test
+
+
+# as_dtype
```
Comment (hirwa-nshuti) [inline ivy_tests/test_ivy/test_frontends/test_torch/test_other_operations.py:9]: Would you change the comment name to clone

File: ivy_tests/test_ivy/test_frontends/test_torch/test_other_operations.py [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+@handle_frontend_test(
+    fn_tree="torch.clone",
+    input_dtype=helpers.get_dtypes("valid", full=False),
+    test_with_out=st.just(False),
+)
+def test_tensorflow_as_dtype(
```
Comment (hirwa-nshuti) [inline ivy_tests/test_ivy/test_frontends/test_torch/test_other_operations.py:15]: This should be renamed to `test_torch_clone`

**Review by hirwa-nshuti — CHANGES_REQUESTED** [review]
Hey @teckno 
Remove the changes to `.idea` files and do check the comments below

File: ivy_tests/test_ivy/test_frontends/test_torch/test_other_operations.py (hunk 3 above)
Comment (teckno) [reply ivy_tests/test_ivy/test_frontends/test_torch/test_other_operations.py:15]: Noted!

File: ivy_tests/test_ivy/test_frontends/test_torch/test_other_operations.py (hunk 2 above)
Comment (teckno) [reply ivy_tests/test_ivy/test_frontends/test_torch/test_other_operations.py:9]: Noted

**Review by KareemMAX — CHANGES_REQUESTED** [review]
Hi @teckno,
Looks like you have changed some files related to the docs, I guess they are not intentional. Can you please revert those changes?

**Pull request closed without merging**